src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,113 @@
+---
+title: CREATE POLICY
+summary: The CREATE POLICY statement defines a new row-level security policy on a table.
+toc: true
+keywords: security, row level security, RLS
+docs_area: reference.sql
+---
+
+The `CREATE POLICY` statement defines a new [row-level security (RLS)]({% link {{ page.version.version }}/row-level-security.md %}) policy on a [table]({% link {{ page.version.version }}/schema-design-table.md %}).
+
+## Syntax
+
+<!--
+
+NB. This is commented out while we wait for a fix to DOC-12125
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/create_policy_stmt.html %}
+</div>
+
+-->
+
+{% include_cached copy-clipboard.html %}
+~~~
+CREATE POLICY [ IF NOT EXISTS ] policy_name ON table_name
+    [ AS { PERMISSIVE | RESTRICTIVE } ]
+    [ FOR { ALL | SELECT | INSERT | UPDATE | DELETE } ]
+    [ TO { role_name | PUBLIC | CURRENT_USER | SESSION_USER } [, ...] ]
+    [ USING ( using_expression ) ]
+    [ WITH CHECK ( check_expression ) ];
+~~~
+
+## Parameters
+
+Parameter | Description
+----------|------------
+`policy_name` | Unique identifier for the policy on the table.
+`table_name` | The [table]({% link {{ page.version.version }}/schema-design-table.md %}) to which the policy applies.
```

Comment:
I don't see a description of `IF NOT EXISTS` in the table.

Reply:
updated to add 

> `IF NOT EXISTS` | Used to specify that the policy will only be created if one with the same `policy_name` does not already exist on `table_name`.